root/lib/systemd/system/titus-imds-proxy@.service

```diff
@@ -23,7 +23,7 @@ LimitNOFILE=65536
 PrivateTmp=yes
 
 EnvironmentFile=-/run/titus-metadata-proxy.salt
-EnvironmentFile=/var/lib/titus-environments/%i/imds-proxy.env
+EnvironmentFile=/var/lib/titus-environments/%i/titus-imds-proxy.env
```

Comment:
Will this be safe to deploy in a mutable way?

Reply:
Should be, it is launched once-per-container.